go.mod

```diff
@@ -13,3 +14,5 @@ require (
 	github.com/libp2p/go-libp2p-routing-helpers v0.1.0
 	github.com/libp2p/go-libp2p-swarm v0.1.0
 )
+
+replace github.com/libp2p/go-libp2p-pubsub v0.1.0 => github.com/aschmahmann/go-libp2p-pubsub v0.0.4-0.20190807152749-d7996289bbcd
```

Comment:
TODO

Reply:
Now uses a version of libp2p/pubsub from master. However, there hasn't been a release yet so it's referencing `v0.1.1-0.20190807100218-9f04364996b4`